### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by srikant96**

### CODE REVIEW

**Comment by amezng** [comment]
Menu needs to be fixed

File: styles/header.css [hunk 1]
```diff
@@ -28,25 +19,94 @@ nav > ul
...
+div.hamburger,.toggle-menu{
+    display:block;
+}
+.topnav>ul{
+    justify-content: start;
+    flex-direction: column;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    right: -300px;
```
Comment (amezng) [inline styles/header.css:95]: No need of this

**Review by amezng — CHANGES_REQUESTED** [review]
Fix css for media query

File: index.html [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
-
+	<nav class="slide">
+           <input type="checkbox" class="toogle-menu">
+                    <div class="hamburger"></div>
 	<ul>
-            <li><a href="">About</a></li>
-            <li><a href="">Articles</a></li>
-            <li><a href="">Project</a></li>
-            <li><a href="">Presentations</a></li>
+            <li><a href="#">About</a></li>
```
Comment (amezng) [inline index.html:21]: Use the section id in the href. For eg. `#about` , `#articles` and so on..

File: styles/header.css [hunk 3]
```diff
@@ -28,25 +19,66 @@ nav > ul
...
     gap: 30px;
+    margin-top: -40px;
 }
 nav >ul> li > a
 {
     padding: 10px;
     color: #666;
     font-weight: bold;
 }
+.hamburger{
```
Comment (amezng) [inline styles/header.css:30]: Remove this code, and use span/div tag.

**Review by amezng — CHANGES_REQUESTED** [review]

File: styles/header.css (hunk 3 above)
Comment (amezng) [reply styles/header.css:30]: set the z-index too

**Comment by srikant96** [comment]
resolved header section - close #26

**Comment by amezng** [comment]
Don't use vs code

**Pull request closed without merging**